scriptRunner/src/twitterImages.js

```diff
@@ -0,0 +1,53 @@
+const fetch = require("node-fetch")
+const database = require("../../api/lib/database")
+const config = require("../../api/config")
+const db = database(config)
+
+module.exports.getTwitterImages = async () => {
+    const networks = await db("").getNetworks()
+    const cosmosNetworks = networks.filter(({network_type}) => network_type === "polkadot")
```

Comment:
shouldn't this be `substrateNetworks` or `polkadotNetworks`?

Reply:
There is a suggestion function :P

Reply:
i wasn't sure which one you'd prefer 😉

Reply:
sure... ;P